src/core/legacy_lexer.mll

```diff
@@ -27,7 +27,7 @@ type token =
   | L_SQB | R_SQB | L_PAR | R_PAR | ARROW | LARROW | FARROW | DEFEQ | COMMA
   | COLON | EQUAL | DOT | EOF
   (* Keywords. *)
-  | KW_DEF | KW_INJ | KW_THM | TYPE | KW_PRV
+  | KW_DEF | KW_INJ | KW_THM | TYPE | KW_PRV | KW_SEQ
```

Comment:
Is it correct to modify the legacy parser? The legacy parser should follow the current syntax of Dedukti.

Reply:
That's a good question. I was thinking that as long as we don't break compatibility, it's ok

Reply:
But who is going to use this syntax? We already have too much work. So, I would prefer not to have more code to maintain if we decide to change things in LP.